exit-status.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: unassigned
+Author: Nicholas Carpenter <ncarpenter@ebsco.com>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Signal outside tools of specific Chef-Client run status.  Ability to determine results of a Chef-Client run.
+
+## Motivation
+    As a Chef user,
+    I want to be able to determine when a chef-client run is rebooting the node,
+    so that Test-Kitchen/Vagrant/any outside tool can wait for node to reboot, and continue converging.
+    
+## Specification
+* Chef-apply, Chef-client, Chef-Solo should honor the below exit chef run exit codes
```

Comment:
Mostly I want to mention utilities here and how they act, but maybe we can also avoid making an inclusive list here? There's an extra use of "exit" in here too.

* Chef applications (e.g. chef-client) that interpret recipes should use the specified exit codes
* Chef tools (e.g. knife) should behave appropriately for the exit code, or pass it to the user